src/config.py

```diff
@@ -8,6 +8,7 @@
 class Ports(Enum):
```

Comment:
Is [SimpleNamespace](https://docs.python.org/3/library/types.html#types.SimpleNamespace) a better fit here?

Reply:
Just to reiterate, an enum seems semantically incorrect here.